dev/reset-shared-store.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
```

Comment:
There's a line in most of our scripts that's needed to make these scripts work in a cygwin environment, due to differences in EOL stuff:

```suggestion
(set -o igncr) 2>/dev/null && set -o igncr; # this comment is required to trick cygwin into dealing with windows vs. linux EOL characters

```